pkg/storage/accessibility_request_document.go

```diff
@@ -121,3 +122,13 @@ func (s *Store) FetchDocumentsByAccessibilityRequestID(ctx context.Context, id u
 
 	return results, nil
 }
+
+// DeleteAccessibilityRequestDocument archives a document
```

Comment:
I worry about this function. If it fails silently, we won't get any information about it. Contrast this with other update functions, which return a fetched version of the data after updating it: if an error occurs in the update, the returned data will still be accurate to what's in the DB.

Reply:
I can't think of anything, and I added a test to check. But you never know!